tests/integration/common.py

```diff
@@ -81,16 +79,51 @@ def return_model(model_id: str, example_model: ExampleModelPyd):
         raise HTTPException(status_code=404, detail="Model not found")
 
 
-@fastapi_test_app.patch("/models/{model_id}")
-async def patch_model(model_id: str, example_model: ExampleModelPyd):
-    return return_model(model_id, example_model)
-
-
-@fastapi_test_app.get("/test_api")
-async def get_test_api():
-    return {"msg": "OK"}
-
-
-@fastapi_test_app.get("/")
-async def get_none():
-    return None
+class CustomResponseHeaders:
```

Comment:
I couldn't think of a better way of getting the configuration into the app without defining it differently every time. Using environment variables for this isn't great, but at least it's all packaged into this single class.